src/lando/headless_api/models/tokens.py

```diff
@@ -1,11 +1,14 @@
 import secrets
+from typing import Self
 
 from django.contrib.auth.hashers import check_password, make_password
 from django.contrib.auth.models import User
 from django.db import models
 
 from lando.main.models.base import BaseModel
 
+API_TOKEN_PREFIX_LENGTH = 8
```

Comment:
Wondering if this should be set to a smaller length. What advantage do we get of setting it to `8` vs say `2` or `4`? Unlikely to cause a huge performance hit given the limited number of users on the platform.